src/controllers/dlgprefcontroller.cpp

```diff
@@ -519,7 +519,7 @@ void DlgPrefController::slotApply() {
 
     QString mappingPath = mappingPathFromIndex(m_ui.comboBoxMapping->currentIndex());
     m_pMapping = LegacyControllerMappingFileHandler::loadMapping(
-            mappingPath, QDir(resourceMappingsPath(m_pConfig)));
+            QFileInfo(mappingPath), QDir(resourceMappingsPath(m_pConfig)));
```

Comment:
Did this slip in by mistake?

Reply:
yes, this was unrelated build fix for Qt6

Reply:
Did they make the corresponding QFileInfo constructor explicit? Otherwise I don't understand why this should be necessary.